documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -265,6 +271,10 @@ suppressGeneratorTimestamp`
 |If set to `true`, the creation of a time stamp in the `@Generated` annotation in the generated mapper classes is suppressed.
 |`false`
 
+|`mapstruct.verbose`
+|If set to `true`, MapStruct in which MapStruct logs its major decisions. Note: also `showWarnings` need to be added due to a problem in the maven-compiler-plugin configuration.
```

Comment:
I think we need to say that `showWarnings` is for use within maven.

Reply:
added additional text